### PR TITLE
fix: Improve action consistency

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

This commit improves consistency and fixes a couple minor UI issues:
- Moves Image > Push and > Show History to kebab menu, making Image list page actions consistent with other pages.
- Adds support for disabled actions in ListItemButtonIcon.
- Makes Image > Delete action disabled (vs missing) so that actions line up and don't change position in the list of images.
- Makes Container Details > Open Browser action disabled (vs missing) so that actions don't change position when container starts.
- Nit: fix imagesUnsubscribe typo in ImagesList

### Screenshot/screencast of this PR

Screenshot 2023-01-23 at 12 11 55 PM

Screenshot 2023-01-23 at 12 09 40 PM

### What issues does this PR fix or reference?

N/A

### CODE REVIEW

**Comment by benoitf** [comment]
it would be nice to have separate commits for all the changes

File: packages/renderer/src/lib/ImagesList.svelte [hunk 1]
```diff
@@ -257,8 +257,8 @@ async function deleteSelectedImages() {
                 <div class="w-full text-right text-sm text-gray-400">{image.humanSize}</div>
               </div>
             </td>
-            <td class="pl-6 text-right whitespace-nowrap rounded-tr-lg rounded-br-lg pr-1">
-              <ImageActions image="{image}" onPushImage="{handlePushImageModal}" />
+            <td class="pl-6 text-right whitespace-nowrap rounded-tr-lg rounded-br-lg">
```
Comment (cdrage) [inline packages/renderer/src/lib/ImagesList.svelte:313]: Good catch! I believe you'll have to modify it too for VolumeList and remove the pr-1? I saw in ContainerList and PodsList we don't use this class, and it's simply a div that floats to the right (maybe we can replace it with: `<div class="text-right w-full">` instead and see if it works?

Just a thought!

**Review by cdrage — CHANGES_REQUESTED** [review]
Very small nitpick, otherwise this LGTM! Thanks for the changes. I agree having the button disabled looks better for consistency as well as tells the user "hey, the feature is there, but you can't use it yet unless X happens".

**Comment by cdrage** [comment]
Screenshot 2023-01-25 at 2 55 04 PM
Screenshot 2023-01-25 at 2 54 57 PM

LGTM for testing it out

File: packages/renderer/src/lib/ImagesList.svelte (hunk 1 above)
Comment (deboer-tim) [reply packages/renderer/src/lib/ImagesList.svelte:313]: Good catch on the volume alignment, I've removed the pr-1 to make add pages aligned. I would prefer not to do the div change as it was already commented that this PR is too big :) and it looks like the container/pod list have actually added two divs to insert in additional status - i.e. we should copy this, but not until we'd put something in the extra div or we do some code consistency cleanup.

**Review by cdrage — APPROVED** [review]

**Comment by benoitf** [comment]
FYI you're not required to rebase/merge from main branch before merging especially when there is no library change
if there are conflicts, it's reported

**Comment by deboer-tim** [comment]
When you do a merge from the web and your account has more than 1 email it prompts which of your addresses to use to sign off the commit. Last time I tried it used a different one anyway and failed DCO. Haven't found any way to configure this or a GitHub bug, so was just going with the safer option for now...

**Comment by benoitf** [comment]
I think the best way is to rebase instead of merge (only if there is a conflict else you don't need to rebase)
and use the squash into one

**Comment by deboer-tim** [comment]
👍🏼 I'll take a better look at the options next time, just got spooked by the "I can't trust which 'me' it's going to use".

**Pull request merged**